.storybook/manager.js

```diff
@@ -39,3 +41,27 @@ addons.register("code/tab", () => {
     render: ({ key, active }) => <>{active && <Playground key={key} />}</>,
   });
 });
+
+addons.register('google-analytics', (api) => {
+  ReactGA.initialize("G-V1N3TQVQB5");
```

Comment:
Should we put it in a environment variable?

Reply:
it's smart not to. GA tracking would only really track when it matches the URL of atlantis.getjobber.com

Reply:
Nice! 

And about this GA code, should we move it to a environment variable or is it okay to keep it public?

Reply:
okay to be public. it's pretty much gonna be public no matter what since it'll show up in the DOM